lib/dpl/provider/elastic_beanstalk.rb

```diff
@@ -27,6 +27,10 @@ def check_auth
       def check_app
       end
 
+      def only_create_app_version
+        options[:only_create_app_version] || false
```

Comment:
`|| false` seems unnecessary, as the value defaults to `nil` is the key doesn't exist.